component/loki/source/file/reader.go

```diff
@@ -0,0 +1,9 @@
+package file
+
+// Reader contains the set of methods the loki.source.file component uses.
+type Reader interface {
+	Stop()
+	IsRunning() bool
+	Path() string
+	MarkPositionAndSize() error
+}
```

Comment:
nit: this doesn't need to be exposed, I don't think.

Reply:
You're right, it doesn't. Fixing this.